### PR TITLE
Add README instructions to configure with Laravel 11

### PR DESCRIPTION
**Pull request by fjahn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ A Laravel package, that allows signing emails with DKIM.
 composer require simonschaufi/laravel-dkim
 ```
 
-The providers array in `config/app.php` has an entry with `Illuminate\Mail\MailServiceProvider::class`. Comment this 
+In **Laravel 9 & 10**, the providers array in `config/app.php` has an entry with `Illuminate\Mail\MailServiceProvider::class`. Comment this
```
Comment (simonschaufi) [inline README.md:18]: How about: In Laravel version below 11?

Comment (fjahn) [reply README.md]: Good Idea!

**Comment by simonschaufi** [comment]
Thank you for your contribution!

**Comment by fjahn** [comment]
> Thank you for your contribution!

Thank *you* for writing and maintaining this awesome package!

**Pull request merged**